### PR TITLE
Fix group-replication cluster creation

### PR DESCRIPTION
**Pull request by pooknull**
In MySQL Shell 8.0.30 `communicationStack` option was added. By default, it uses the `MYSQL` communication stack value, which doesn't allow us to use the `ipAllowList` option.

As mentioned here: https://dev.mysql.com/doc/relnotes/mysql-shell/8.0/en/news-8-0-30.html, the `MYSQL` communication stack obsoletes the usage of IP allowlists. So, they should be removed.

Also, fixes the `mysqlsh` warning `Cannot set LC_ALL to locale en_US.UTF-8: No such file or directory` by installing `glibc-langpack-en`

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -17,47 +17,48 @@ COPY . .
...
+    && curl -Lf -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
+    && rpmkeys --checksig /tmp/percona-release.rpm \
+    && rpm -i /tmp/percona-release.rpm \
+    && rm -rf "$GNUPGHOME" /tmp/percona-release.rpm \
+    && rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY \
+    && percona-release setup pdps-8.0
 
 RUN set -ex; \
     microdnf update; \
+    microdnf install -y glibc-langpack-en; \
```
Comment (egegunes) [inline build/Dockerfile:61]: you may consider doing it in a single `microdnf install`

**Review by egegunes — DISMISSED** [review]
one unimportant comment, up to you. LGTM

**Review by inelpandzic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/init-deploy.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/auto-config.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/demand-backup.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/service-per-pod.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/config.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/limits.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/version-service.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/one-pod.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/gr-demand-backup.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/tls-cert-manager.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/scaling.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-224/cf3b304/users.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-server-mysql-operator/pull/224/commits/cf3b304f896172e108f81fa6c50c926911462508
image: `perconalab/percona-server-mysql-operator:PR-224-cf3b304`

File: build/Dockerfile (hunk 1 above)
Comment (hors) [reply build/Dockerfile]: [16aab73](https://github.com/percona/percona-server-mysql-operator/pull/224/commits/16aab7318cb0c548e759f4368ce2bb6ad33dae12)

**Pull request merged**